src/lib/features/client-feature-toggles/client-feature-toggle.controller.ts

```diff
@@ -407,4 +399,14 @@ export default class FeatureController extends Controller {
         const jsonString = JSON.stringify(value);
         return Buffer.byteLength(jsonString, 'utf8');
     }
+
+    deepEqualIgnoreOrder = (obj1, obj2) => {
```

Comment:
types seem to be missing here and this function doesn't need to be attached to class